ecs-cli/modules/commands/cluster/cluster_command.go

```diff
@@ -26,7 +26,7 @@ func UpCommand() cli.Command {
 		Usage:  "Creates the ECS cluster (if it does not already exist) and the AWS resources required to set up the cluster.",
 		Before: ecscli.BeforeApp,
 		Action: cluster.ClusterUp,
-		Flags:  append(clusterUpFlags(), command.OptionalClusterFlag(), command.OptionalRegionFlag()),
+		Flags:  append(append(clusterUpFlags(), command.OptionalRegionFlag()), command.OptionalClusterAndProfileFlags()...),
```

Comment:
The reason why `OptionalClusterFlags` and `OptionalRegionFlags` were separated was that ECR only needs region flags. I do agree that we need a better way to express this.